scripts/fig2.py

```diff
@@ -2,10 +2,10 @@
 from common import label
 
 Figure(
-    "23.1cm", "6cm",
+    "28cm", "6cm",
```

Comment:
Sounds, a bit too wide. We only have letter width.

Reply:
whoops, OK I'll readjust